Packs/PAN-OS/Integrations/Panorama/Panorama.yml

```diff
@@ -4614,8 +4614,286 @@ script:
     - contextPath: Panorama.Content.Install.Status
       description: Installation status.
       type: string
-    description: Installs specific content update file.
-  dockerimage: demisto/python3:3.9.5.20070
+  - arguments:
+    - default: false
+      defaultValue: null
+      description: String to filter to only show specific hostnames or serial numbers.
+      isArray: false
+      name: device_filter_string
+      required: false
+      secret: false
+    deprecated: false
+    description: Gets all arp tables from all firewalls in the topology.
+    name: pan-os-platform-get-arp-tables
+    outputs:
+    - contextPath: PANOS.ShowArp.Summary.hostid
+      description: ID of PAN-OS host
+      type: string
+    - contextPath: PANOS.ShowArp.Summary.max
+      description: Maximum supported ARP Entries
+      type: string
+    - contextPath: PANOS.ShowArp.Summary.total
+      description: Total current arp entries
+      type: string
+    - contextPath: PANOS.ShowArp.Summary.timeout
+      description: ARP entry timeout
+      type: string
+    - contextPath: PANOS.ShowArp.Summary.dp
+      description: Firewall dataplane associated with Entry
+      type: string
+    - contextPath: PANOS.ShowArp.Result.hostid
+      description: ID of PAN-OS host
+      type: string
+    - contextPath: PANOS.ShowArp.Result.interface
+      description: Network interface learnt ARP entry
+      type: string
+    - contextPath: PANOS.ShowArp.Result.ip
+      description: layer 3 address
+      type: string
+    - contextPath: PANOS.ShowArp.Result.mac
+      description: Layer 2 address
+      type: string
+    - contextPath: PANOS.ShowArp.Result.port
+      description: Network interface matching entry
+      type: string
+    - contextPath: PANOS.ShowArp.Result.status
+      description: ARP Entry status
+      type: string
+    - contextPath: PANOS.ShowArp.Result.ttl
+      description: Time to Live
+      type: string
+  - arguments:
+    - default: false
+      defaultValue: null
+      description: String to filter to only show specific hostnames or serial numbers.
+      isArray: false
+      name: device_filter_string
+      required: false
+      secret: false
+    deprecated: false
+    description: Pulls all route summary information from the topology
+    name: pan-os-platform-get-route-summary
+    outputs:
+    - contextPath: PANOS.ShowRouteSummary.Summary.hostid
+      description: ID of PAN-OS host
+      type: number
```

Comment:
I am sorry maybe I misled you, but those need to be case sensitive. These are the options: 
![image](https://user-images.githubusercontent.com/53861351/154674920-9b0ae279-9c71-4cd6-b68e-ed6300722f1e.png)

Please change that for all the context outputs types

Reply:
Done